tests/backends/test_numba_object.py

```diff
@@ -1390,17 +1390,11 @@ def get_pos(v):
 def test_operator_truth():
     @numba.njit
     def get_true(v):
-        if v:
-            return True
-        else:
-            return False
+        return bool(v)
 
     @numba.njit
     def get_false(v):
-        if not v:
-            return True
-        else:
-            return False
+        return not bool(v)
```

Comment:
If not, it should be written `return True if v else False`, and  `return False if v else True` if this is not allowed. But hopefully, it is.

Reply:
The question I had in mind was not whether Numba can do it at all with any data types (it can), but one of these capabilities is defined in terms of the other for this specific type. This might be part of the sausage-making that allows user code in Numba to look like ordinary, fully featured Python.

Reply:
Do you know if there's a test? Given this is _in_ a test, I would assume so, but maybe we should have one of each? A test like this, and a test with the if/else, since they should lower/type differently?

Reply:
Added a test for `bool(v)` and an `if x` test. Not sure we need a "not" version, as that's just a normal not in numba once this is truthy, would save a JIT compile. Ehh, more tests is better.